### ISSUE TITLE
An error during module loading

### ISSUE THREAD

**Issue by daeyeon** [Samsung/iotjscode#89]
I observe the following error.

Pre-condition:

1. There is any module in `iotjs_modules`. e.g) `iotjs_modules/mymodule`.
2. user entry js tries loading `mymodule` through `var mymodule = require('mymodule');`

Observation:

The above code locally works, but an error occurs during remote-debugging.
It's seemingly related to the native call inside ` fs.statSync` which is used in `module.js`.

Can you clarify the above observation is right?

**Comment by knightburton** [comment]
Well, I tested your problem with two different way:

1. Required a built-in module.
   I wrote a little test file which is included the `http` module and called the `createServer` function.
   ```javascript
   var http = require('http');
   
   http.createServer(function(req, res) {
     // code here....
   });
   ```
   Then I pushed this into the iotjscode with iotjs (I built the iotjs in debug and release, too): `./build/x86_64-linux/release/bin/iotjs --start-debug-server test.js`

   This test is worked well.

2. Required a new user defined module.
I wrote a new module into the iotjs, based on [this documentation](https://github.com/Samsung/iotjs/blob/master/docs/devs/Writing-New-Module.md). Then I pused the `user.js` into the iotjscode as before. This version if worked, too.

I hope that I understood what are you trying to achieve.
